content/en/docs/architecture/branching.md

```diff
@@ -10,42 +10,30 @@ The number of repositories and branches involved in assembling an OpenShift rele
 
 ## Normal Development For 4.x Release
-During normal development, active development branches are open to all changes.
+The vast majority of the time active development branches are open to all changes.
 
 |*Branch*|*Promotes To*|*Merge Criteria*|*Fast-Forwarded?*|
 |-|-|-|-|
 |`master/main`|`ocp/4.x`|`lgtm`, `approved`|no|
-|`release-4.x-n`|`ocp/4.x-n`|`lgtm`, `approved`, `bugzilla/valid-bug`, `cherry-pick-approved`|no|
+|`release-4.x-n`|`ocp/4.x-n`|`lgtm`, `approved`, `jira/valid-bug`, `cherry-pick-approved`|no|
 |`release-4.x`|nowhere|merges blocked|yes|
 |`release-4.x+1`|`ocp/4.x+1`|merges blocked|yes|
 
-## Feature Freeze For 4.x Release
-During feature freeze, the expectation is that merges to the active development branch are fixing bugs, so we require that all PRs that land have a valid bug label.
+The primary exception to this rule is when [the Technical Release Team](../release-oversight/the-technical-release-team/) decides to enable the `/acknowledge-critical-fixes-only` label.
 
-|*Branch*|*Promotes To*|*Merge Criteria*|*Fast-Forwarded?*|
-|-|-|-|-|
-|`master/main`|`ocp/4.x`|`lgtm`, `approved`, `bugzilla/valid-bug`|no|
-|`release-4.x-n`|`ocp/4.x-n`|`lgtm`, `approved`, `bugzilla/valid-bug`, `cherry-pick-approved`|no|
-|`release-4.x`|nowhere|merges blocked|yes|
-|`release-4.x+1`|`ocp/4.x+1`|merges blocked|yes|
-
-### No-Feature-Freeze Process Pilot
-
-Some teams are participating in a pilot of a different process where Feature Freeze phase does not affect merge criteria on the `master/main` branch. Repositories that belong to these teams merge pull requests to their `master/main` branches if they are labelled with one of the following sets of labels:
-
-- `lgtm`, `approved`, `bugzilla/valid-bug` (for bugfix pull requests)
-- `lgtm`, `approved`, `px-approved`, `docs-approved`, `qe-approved` (for feature pull requests)
-
-These criteria stay the same in both normal development and feature freeze phases (as opposed to standard process where such PRs require `bugzilla/valid-bug` during feature freeze and not during normal development). Pull requests to `release-*` branches are not affected by this process. They use the criteria specified in the respective sections of this document.
-
-## Code Freeze For 4.x Release
-During code freeze, the active development branch opens to contributions that target the future release. Contributions to the release branch require staff engineer approval as they’re landing so close to the moment we are publishing the release publicly.
+## Final Stabilization For Soon-To-Be-Released Minor Versions
+While we no longer have official "code freeze windows, the final two weeks
+before a minor release becomes generally available are still special.  Any
+delays during this period cause costly communication overhead.  As such, we
+require Staff Engineers to sign off on code that will be backported to make
+sure it's essential for the first generally available release and not simply
```

Comment:
don't we also sometimes land things in the about-to-GA branch because they're associated with warm bugs on older 4.y, to preserve the backport chain's continuity?  But even if we do, perhaps that is too hard/distracting to explain here?

Reply:
I think it's fair to mention that case too.  Essentially, we just want to make sure whatever is being merged can't be delayed a week or two.

Reply:
Yes, backports through the to be GA'd version are common, if not the majority of the things that arise during this window of time.